### DISCUSSION TITLE
Extra Network: Loras not showing up

### DISCUSSION THREAD

**Discussion by Eleksar387** [vladmandic/sdnext#431] [Q&A]
If I open the Lora tab in the extra networks none of my Loras showing up. The error message is
```
Nothing here. Add some content to the following directories:
C:\Users\xxx\stable-diffusion-webui\models\Lora
```
But there are Lora models in this path. It is the same path as for automatic1111

**Comment by tkiikt** [comment]
What paths are specified in System Paths in Settings tab and what is the version? The latest version should be fine.

**Comment by Eleksar387** [comment]
After a restart is was there. I assumed there was a bug because it worked fine for checkpoints. Anyway. All good. Thanks!
